src/thor/dao/task_dao.py

```diff
@@ -34,154 +35,150 @@ def session_scope():
         session.close()
 
 
-def manCreateTask(id, name, status, release_id):
+def manual_create_task(key, name, status, release_id):

```

Comment:
Probably a TODO; some of these look like the release_dao functionality and might be consolidable. However, if we expect these 2 files to start diverging from each other in the future, this is fine.

Reply:
Well observed.
Both functions that allow the operator to manually create a release and a task contain almost identical code.

We can leverage Polymorphism here @colinyaos with an Abstract DAO class and make Release DAO and Task DAO inherit from the parent Generic DAO class.
Then the Fast API layer will call one of the instances of the Generic DAO that can just instantiate the object (either `Release` or `Task` and call something like:
```
super().manual_create_entry(release_instance)
or
super().manual_create_entry(task_instance)
```
Depending on the dao module function that is executed.

But, as @brianwu-cdis mentioned, that could be a TODO, it is not necessarily critical right now.